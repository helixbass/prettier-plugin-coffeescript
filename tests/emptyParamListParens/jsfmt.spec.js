run_spec(__dirname, ['coffeescript'], {emptyParamListParens: true})
run_spec(__dirname, ['coffeescript'], {emptyParamListParens: false})
