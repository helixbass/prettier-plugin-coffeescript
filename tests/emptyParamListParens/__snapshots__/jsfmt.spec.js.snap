// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`functions.coffee 1`] = `
() ->

->

=>

class A
  b: ->
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
() ->

() ->

() =>

class A
  b: () ->

`;

exports[`functions.coffee 2`] = `
() ->

->

=>

class A
  b: ->
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
->

->

=>

class A
  b: ->

`;
